package.json

```diff
@@ -16,11 +16,14 @@
         "hoek": "3.x.x"
     },
     "devDependencies": {
-        "lab": "7.x.x"
+        "lab": "8.x.x",
+        "markdown-toc": "^0.12.3"
```

Comment:
Can't guarantee it'll keep working with a 0.x version, this should be made sticky to 0.12 as joi did.